sdv/data_processing/data_processor.py

```diff
@@ -567,21 +567,23 @@ def _create_config(self, data, columns_created_by_constraints):
                     )
                     sdtypes[column] = 'text'
 
-                elif column in self._keys:
-                    prefix = None
-                    if not is_numeric:
-                        prefix = 'sdv-id-'
+                else:
+                    bothify_format = 'sdv-id-??????'
```

Comment:
This will only generate `a-zA-Z` characters. If we want to use `0-9a-zAZ` we could pass `string.ascii_letters + string.digits` as function_kwargs.
Here is an example:
<img width="973" alt="image" src="https://github.com/sdv-dev/SDV/assets/41479552/d5691c2d-f58e-4c62-9928-a507541c8397">

Reply:
After speaking with @npatki I think we can leave as is